src/tldr.ts

```diff
@@ -139,6 +123,23 @@ const completionSpec: Fig.Spec = {
       name: ["-c", "--clear-cache"],
       description: "Delete the entire local cache",
     },
+    {
+      name: "-p",
+      description: "Select platform",
+      args: {
+        name: "platform",
+        suggestions: ["linux", "osx", "sunos", "windows", "common"],
+      },
+    },
+    {
+      name: "--platform",
+      description: "Select platform",
+      insertValue: "--platform={cursor}",
```

Comment:
Maybe we can use `requiresSeparator` instead.
```suggestion
      requiresSeparator: true,
```
https://github.com/withfig/autocomplete-tools/blob/main/types/index.d.ts#L713-L725